### PR TITLE
[EAK-330] Website redesign 2022

### PR DESCRIPTION
**Pull request by ala-n**
Draft PR for site redesign

- [x] CI actions updated on main (waiting for #343)
- [x] dependencies updated (@abarmina)
- [x] linting process updated (@ala-n )
- [x] code cleanup (@abarmina, @ala-n )
- [x] code verified (@abarmina, @ala-n, @NastaLeo, @bhlynsky, @Bogdan9817 )

---

- [x] **READY FOR REVIEW**
- [x] **READY FOR MERGE**

Closes: #330

### CODE REVIEW

File: docs/content/dev-tools/depends-on/usage.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <!--
 layout: content
 title: Usage
+seoTitle: Usages - Exadel Authoring Kit
```
Comment (NastaLeo) [inline docs/content/dev-tools/depends-on/usage.md:4]:
```suggestion
seoTitle: Usage - Exadel Authoring Kit
```

File: docs/website/src/eak-zoom-image/eak-zoom-image.less [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    background-size: 16px 16px;
+    background-position: center;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 384 512'%3E%3Cpath fill='%23ffffff' d='M376.6 427.5c11.31 13.58 9.484 33.75-4.094 45.06c-5.984 4.984-13.25 7.422-20.47 7.422c-9.172 0-18.27-3.922-24.59-11.52L192 305.1l-135.4 162.5c-6.328 7.594-15.42 11.52-24.59 11.52c-7.219 0-14.48-2.438-20.47-7.422c-13.58-11.31-15.41-31.48-4.094-45.06l142.9-171.5L7.422 84.5C-3.891 70.92-2.063 50.75 11.52 39.44c13.56-11.34 33.73-9.516 45.06 4.094L192 206l135.4-162.5c11.3-13.58 31.48-15.42 45.06-4.094c13.58 11.31 15.41 31.48 4.094 45.06l-142.9 171.5L376.6 427.5z'/%3E%3C/svg%3E");
+    background-repeat: no-repeat;
+
+    border-radius: 50%;
+    background-color: gray;
+
+    @media @xs {
```
Comment (NastaLeo) [inline docs/website/src/eak-zoom-image/eak-zoom-image.less:44]: do we can follow mobile-first?

File: docs/website/src/eak-zoom-image/eak-zoom-image.ts [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  @boolAttr()
+  public inZoom: boolean;
+  @boolAttr()
+  public inDrag: boolean;
+
+  protected startPoint: Point;
+  protected startPosition: Point;
+
+  @memoize()
+  get $image() {
```
Comment (NastaLeo) [inline docs/website/src/eak-zoom-image/eak-zoom-image.ts:25]:
```suggestion
  get $image(): HTMLElement {
```

File: docs/website/src/eak-zoom-image/eak-zoom-image.ts [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    const originalImage = document.createElement('esl-image');
+    originalImage.setAttribute('data-src', this.src);
+    originalImage.setAttribute('lazy', '');
+    originalImage.setAttribute('mode', 'fit');
+    return originalImage;
+  }
+
+  @memoize()
+  get $closeButton() {
```
Comment (NastaLeo) [inline docs/website/src/eak-zoom-image/eak-zoom-image.ts:34]:
```suggestion
  get $closeButton(): HTMLElement {
```

File: docs/website/src/landing/article-section/article-section.less [hunk 5]
```diff
@@ -1,119 +1,150 @@
...
+
+  &-second {
+    grid-area: second;
+  }
+
+  &-third {
+    grid-area: third;
+  }
+
+  &-forth {
```
Comment (NastaLeo) [inline docs/website/src/landing/article-section/article-section.less:139]:
```suggestion
  &-fourth {
    grid-area: fourth;
```

File: docs/website/src/landing/article-section/article-section.less [hunk 6]
```diff
@@ -1,119 +1,150 @@
...
+    grid-area: fouth;
```
Comment (NastaLeo) [inline docs/website/src/landing/article-section/article-section.less:140]:
```suggestion
    grid-area: fourth;
```

File: docs/website/src/navigation/sidebar/sidebar-seccondary-nav.less [hunk 7]
```diff
@@ -1,101 +1,91 @@
...
-     padding-left: 13px;
-    }
+
     .sidebar-nav-secondary .sidebar-nav-secondary-list .sidebar-nav-item-trigger {
-      padding-left: 26px;
+      padding-left: 13px;
+      @media @sm-md {
+        padding-left: 15px;
+      }
+      @media @md-xl {
```
Comment (NastaLeo) [inline docs/website/src/navigation/sidebar/sidebar-seccondary-nav.less:54]: we can remove it be as is identical to default value

**Review by fshovchko — APPROVED** [review]

File: docs/website/views/index.11tydata.yml [hunk 8]
```diff
@@ -10,20 +10,22 @@ landing:
...
 
   features:
     title: What’s the Exadel Authoring Kit for AEM all about?
     items:
-      - A smart thoroughly tested extension for AEM authoring interfaces
       - Meant for developers, authors, and business analysts working in an AEM environment
-      - Filled with reusable coding and visual patterns
       - Rich in utilities for handling AEM Sling models, modifying AEM component policy,
-        and managing structured data
+          and managing structured data
```
Comment (fshovchko) [inline docs/website/views/index.11tydata.yml:24]:
```suggestion
        and managing structured data
```
Unnecessary whitespace

File: docs/website/views/_layouts/popular-articles.njk [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+      </li>
+    {# Search for articles only using its link (/content/..) #}
+    {% else %}
+    {% set articleLink = article %}
+      {% for item in collections.all | filterArticles(articleLink) %}
+        <li class="popular-articles-item">
+          {% set text = item.data.content %}
+          <h4 class="mb-2">{{ text | articleHeader | striptags }}</h4>
+          <p class="popular-articles-content">{{ text | articleContent | striptags }}</p>
+          <a href="{{ item.data.page.filePathStem | url }}"> Learn More</a>
```
Comment (fshovchko) [inline docs/website/views/_layouts/popular-articles.njk:24]:
```suggestion
          <a href="{{ item.data.page.filePathStem | url }}">Learn More</a>
```
Unnecessary whitespace

**Review by liubou-masiuk — APPROVED** [review]

**Review by mhoroshko — APPROVED** [review]

File: docs/website/11ty/markdown.shortcut.js [hunk 10]
```diff
@@ -8,12 +8,12 @@ const {github, rewriteRules, urlPrefix} = require('./site.config');
 
 const recursiveCheckLinks = (arr, link, element, key) => {
     arr.forEach(el=>{
-        if(el.hasOwnProperty("fileName") && el.fileName === link){
-            element.setAttribute("href", `/${[key]}/${link.replace(".md", "")}`);
+        if(el.hasOwnProperty('fileName') && el.fileName === link){
```
Comment (yadamskaya) [inline docs/website/11ty/markdown.shortcut.js:11]:
```suggestion
        if (el.hasOwnProperty('fileName') && el.fileName === link) {
```

**Review by NastaLeo — APPROVED** [review]

File: docs/website/11ty/resolve-path.transform.js [hunk 11]
```diff
@@ -32,11 +33,16 @@ class PathResolver {
...
+
   static resolveImages(dom, filePath) {
-    dom.querySelectorAll('img[src^="."]').forEach((img) => {
+    dom.body.querySelectorAll('img[src^="."]').forEach((img) => {
       const resolved = PathResolver.resolveLink(img.src, filePath);
       if (resolved !== img.src) console.info(blue(`Rewrite image source "${img.src}" to "${resolved}"`));
-      img.src = resolved;
+
+      const zoomImg = dom.createElement('eak-zoom-image')
+      zoomImg.setAttribute('data-src', resolved)
```
Comment (yadamskaya) [inline docs/website/11ty/resolve-path.transform.js:43]:
```suggestion
      const zoomImg = dom.createElement('eak-zoom-image');
      zoomImg.setAttribute('data-src', resolved);
```

**Review by bhlynsky — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by smiakchilo — APPROVED** [review]

**Review by abarmina — APPROVED** [review]
